astropy/stats/sigma_clipping.py

```diff
@@ -144,10 +144,19 @@ def _perform_clip(self, _filtered_data, axis=None):
         min_value = max_value - std * self.sigma_lower
         max_value += std * self.sigma_upper
 
+        # Ensure min/max can be broadcast with the data (if arrays):
         if axis is not None:
-            if axis != 0:
-                min_value = np.expand_dims(min_value, axis=axis)
-                max_value = np.expand_dims(max_value, axis=axis)
+            if not hasattr(axis, '__iter__'):
+                axis = (axis,)
+
+            # Convert negative indices & restore reduced axes, with length 1:
+            axis = tuple(_filtered_data.ndim + n if n < 0 else n for n in axis)
+            mshape = tuple(1 if dim in axis else sz \
```

Comment:
Remove the backslash to get rid of the warning. :smile:

Reply:
Yep, gone.